### PR TITLE
[FIX] Some fixes and improvements in l10n_cl module

### PR DESCRIPTION
**Pull request by Danisan**
[FIX] add account asignation to taxes
[FIX] Fixes after code review
[FIX] remove default value for taxpayer_type
[IMP] add an empty section in res_config_settings_view as a holder
for any Chilean Localization future configuration setting.

Description of the issue/feature this PR addresses:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_cl/views/res_config_settings_view.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field name="name">res.config.settings.view.form.chilean.loc</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="account.res_config_settings_view_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//div[@id='taxcloud_settings']//parent::*" position="after">
```
[the remark this answers is not in the document]
Comment (Danisan) [reply addons/l10n_cl/views/res_config_settings_view.xml]: read. Working on it

[the remark this answers is not in the document]
Comment (Danisan) [reply addons/l10n_cl/views/res_config_settings_view.xml]: Fixed in 8dfaeae. Not squashed over this already pushed PR

**Comment by jco-odoo** [comment]
@robodoo r+

**Comment by d-fence** [comment]
robodoo retry

**Pull request closed without merging**